flow/scripts/util.tcl

```diff
@@ -99,3 +99,19 @@ proc find_macros {} {
   }
   return $macros
 }
+
+proc erase_non_stage_variables {stage_name} {
+  # "$::env(SCRIPTS_DIR)/stage_variables.py stage_name" returns list of
+  # variables to erase.
+  # 
+  # Tcl yaml package can't be imported in the sta/openroad environment:
+  # 
+  # https://github.com/The-OpenROAD-Project/OpenROAD/issues/5875
+  set variables [exec $::env(SCRIPTS_DIR)/stage_variables.py $stage_name]
```

Comment:
This refers to stage_variables.py but I only see flow/scripts/variables.py in this PR.

Reply:
oops, force pushed fix to remove it.